core/controllers/learner_dashboard_test.py

```diff
@@ -333,3 +371,57 @@ def test_anonymous_feedback_is_recorded_correctly(self):
 
         self.assertEqual(messages_summary['author_username'], None)
         self.assertEqual(messages_summary['author_picture_data_url'], None)
+
+    def test_get_suggestions_after_updating_suggestion_summary(self):
```

Comment:
Could you please add some spacing (like you did above) to separate the different steps in testing?

Reply:
Is it OK now?

Reply:
Much much better :) thanks!